packages/venia-concept/src/actions/cart/__tests__/asyncActions.spec.js

```diff
@@ -922,12 +952,18 @@ describe('toggleCart', () => {
 
     test('its thunk opens the drawer and refreshes the cart', async () => {
         await toggleCart()(...thunkArgs);
+        // toggleDrawer
+        expect(dispatch).toHaveBeenNthCalledWith(1, expect.any(Function));
+        expect(dispatch).toHaveBeenCalledTimes(1);
```

Comment:
Because you haven't mocked `getState` here, the call to `toggleCart` on line 954 will use the default mock return value declared at the top of the file:

```js
beforeAll(() => {
    getState.mockImplementation(() => ({
        app: { drawer: null },
        cart: { cartId: 'CART_ID' },
        user: { isSignedIn: false }
    }));
});
```

which makes these assertions a duplicate of your previous test case "its thunk opens the drawer when the cart has no details" (note that the `cart` above has no `details`). Therefore you can remove lines 954-957.

Reply:
I've tried to fixed it.please review and let me know.